packages/markdown/remark/src/types.ts

```diff
@@ -37,7 +37,9 @@ export type ThemePresets = BuiltinTheme | 'css-variables';
 
 export interface ShikiConfig
 	extends Pick<CreateShikiHighlighterOptions, 'langs' | 'theme' | 'themes' | 'langAlias'>,
-		Pick<ShikiHighlighterHighlightOptions, 'defaultColor' | 'wrap' | 'transformers'> {}
+		Pick<ShikiHighlighterHighlightOptions, 'defaultColor' | 'wrap' | 'transformers'> {
+	excludeLangs?: string[];
```

Comment:
I am a bit concerned about the placement of this new option. Shiki doesn't have a `excludeLangs` option, and `shikiConfig` is an object that **must match** the shiki config options.

Wouldn't adding the option to `markdown` make more sense? Of course, we would need to change the logic of the `prism` syntax highlighter, too.

Reply:
Updated. Let me know what you think of this change.